pkg/agent/controller/networkpolicy/cache.go

```diff
@@ -863,6 +870,11 @@ func (c *ruleCache) GetCompletedRule(ruleID string) (completedRule *CompletedRul
 
 	r := obj.(*rule)
 
+	if !c.antreaProxyEnabled && r.Direction == v1beta.DirectionOut && len(r.To.ToServices) != 0 {
+		klog.V(2).InfoS("Field `toServices` can be only used when AntreaProxy is enabled", "ruleID", ruleID)
+		return nil, false, false
+	}
```

Comment:
It still seems to me that this could be considered a bug fix, and hence should be in a dedicated PR?

Reply:
I'll make another PR to add this part of code.